content/docs/guides/logical-replication-cloud-sql.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Replicate data from Cloud SQL Postgres
+subtitle: Learn how to replicate data from Google Cloud SQL Postgres to Neon
+enableTableOfContents: true
+isDraft: false
+updatedOn: '2024-08-02T17:25:18.435Z'
+---
+
+The guide describes how to replicate data using native Postgres logical replication, as described in [Set up native PostgreSQL logical replication](https://cloud.google.com/sql/docs/postgres/replication/configure-logical-replication#set-up-native-postgresql-logical-replication), in the Google Cloud SQL documentation.
+
+## Prerequisites
+
+- A Cloud SQL Postgres instance containing the data you want to replicate. If you need some data to play with, you can use the following statements to create a table with sample data. Your database and schema may differ.
+
+  ```sql shouldWrap
+  CREATE TABLE IF NOT EXISTS playing_with_neon(id SERIAL PRIMARY KEY, name TEXT NOT NULL, value REAL);
+  INSERT INTO playing_with_neon(name, value)
+  SELECT LEFT(md5(i::TEXT), 10), random() FROM generate_series(1, 10) s(i);
+  ```
+
+  <Admonition type="note">
+  This guide uses the default `postgres` data and `public` schema in the Cloud SQL Postgres instance. Your database and schema may differ, but the same steps should apply.
+  </Admonition>
+
+- A Neon project with a Postgres database to receive the replicated data. For information about creating a Neon project, see [Create a project](/docs/manage/projects#create-a-project).
+
+## Prepare your Cloud SQL source database
+
+This section describes how to prepare your source Cloud SQL Postgres instance (the publisher) for replicating data to Neon.
+
+### Enable logical replication
+
+The first step is to enable logical replication at the source Postgres instance. In Cloud SQL, you can enable logical replication for your Postgres instance by setting the `cloudsql.logical_decoding` flag to `on`. This action will set the Postgres `wal_level` parameter to `on`.
```

Comment:
shouldn't it be `wal_level` to `logical`?

Reply:
yes! TY